app/src/main/java/io/github/androidacademyglobal/profile/Previews.kt

```diff
@@ -0,0 +1,62 @@
+package io.github.androidacademyglobal.profile
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.tooling.preview.Preview
+import io.github.androidacademyglobal.R
+import io.github.androidacademyglobal.ui.theme.AndroidAcademyAppTheme
+
+val testProfileScreenState: ProfileScreenState = ProfileScreenState(
+    userId = "userId",
+    photo = R.drawable.ic_launcher_background,
+    displayName = "Grinya",
+    email = "test@google.com",
+    telegram = "GMachine"
+)
+
+@Preview(widthDp = 340, name = "340 width - Me")
+@Composable
+fun ProfilePreview340() {
```

Comment:
u can use less {} 
with fun ProfilePreview340() =
for all cases below

Reply:
fixed